spec/models/html_publication_spec.rb

```diff
@@ -66,4 +66,35 @@
       expect(html_publication.withdrawn_explanation).to eq("because of a reason")
     end
   end
+
```

Comment:
Again, there's a shared example for this: https://github.com/alphagov/frontend/blob/main/spec/models/call_for_evidence_spec.rb#L10